analyses/cell-type-wilms-tumor-06/00_run_workflow.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+# This script runs the module workflow.
+#
+# USAGE:
+# bash 00_run_workflow.sh
+#
+# USAGE in CI:
+# TESTING=1 bash 00_run_workflow.sh
+
+set -euo pipefail
+
+IS_CI=${TESTING:-0}
+project_id="SCPCP000006"
+
+# Ensure script is being run from its directory
+module_dir=$(dirname "${BASH_SOURCE[0]}")
+cd ${module_dir}
+
+# Define directories
+data_dir="../../data/current"
+notebook_template_dir="notebook_template"
+notebook_output_dir="notebook"
+
+
+# Download and create the fetal kidney reference (Stewart et al)
+Rscript scripts/download-and-create-fetal-kidney-ref.R
+
+# Build the gene position file reference for infercnv
+Rscript scripts/06a_build-geneposition.R
```

Comment:
It is confusing to have this out of numerical order. Do we want to rename the file or move this step?

Reply:
I agree. I moved it at one point, and then in the spirit of keeping things as they were, I unmoved it. But you see it too, so yes let's move it.